.github/workflows/package.yml

```diff
@@ -74,6 +74,28 @@ jobs:
       pytestArgs: tests/unit
     secrets: inherit
 
+  run_end2end_tests:
+    strategy:
+      matrix:
+        include:
+          - searchEngineDockerImage: docker.elastic.co/elasticsearch/elasticsearch:8.8.2
+            searchEngineDockerEnv: '{"discovery.type": "single-node", "xpack.security.enabled": "false"}'
+            coverageReport: coverage-elasticsearch-8.8.2
+            runsOn: extended-runner
+          - searchEngineDockerImage: opensearchproject/opensearch:2.4.1
+            searchEngineDockerEnv: '{"discovery.type": "single-node", "plugins.security.disabled": "true"}'
+            coverageReport: coverage-opensearch-2.4.1
+            runsOn: ubuntu-latest
+    name: Run end2end tests
+    uses: ./.github/workflows/end2end-examples.yml
+    needs: check_repo_files
+    if: needs.check_repo_files.outputs.end2endChanges == 'true'
```

Comment:
We can start with this condition, but these tests won't be executed all the time. We can schedule a nightly execution on dev, and one execution when tag/release is going to happen.

Reply:
👍